src/Storage/Storage.Management/help/Copy-AzStorageBlob.md

```diff
@@ -255,7 +255,9 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Force to overwrite the existing blob or file
+Answers 'yes' to any confirmation prompts.
+Force 'yes' to overwrite the existing blob or file.
+Force 'yes' to "Can't get source blob Tags, so source blob tags won't be copied to dest blob. Do you want to continue the blob copy?" when there is an HTML status error 401, 403, or 404 when reading Tags from the blob or file.
```

Comment:
With `-Force`, we will not input 'yes', but just skip the prompt.
And `-Force` will suspress all error codes in get blob tag, but not only 401,403 or 404.

So a better description might be:

> Force to overwrite the existing blob or file.
> Force to continue the copy operation, even can't get blob tag from source blob.